efel/cppcore/LibV5.cpp

```diff
@@ -571,7 +571,7 @@ static int __spike_width1(const vector<double>& t, const vector<double>& v,
     delta_v = v[fall_index] - v[fall_index - 1];
     delta_t = t[fall_index] - t[fall_index - 1];
     t_dev_fall = delta_t * v_dev / delta_v;
-    spike_width1.push_back(t[fall_index] + t_dev_rise - t[rise_index] +
+    spike_width1.push_back(t[fall_index] - t_dev_rise - t[rise_index] +
```

Comment:
This change changes the output of this feature, and that makes the tests fail, since they do not get what they are expected.
@wvangeit are you ok with this fix and should I change the tests expectations accordingly?

Reply:
@AurelienJaquier, this should be handled in a separate PR. So that we can bump the version of eFEL at this change, since it will affect the results. Maybe you can first do the doc, and then this fix?

Reply:
Right. I removed the feature change. Only the doc changes remain.